docs/howto/operator/gcp/analytics.rst

```diff
@@ -48,3 +48,18 @@ To list accounts from Analytics you can use the
 
 You can use :ref:`Jinja templating <jinja-templating>` with
 :template-fields:`airflow.providers.google.marketing_platform.operators.analytics.GoogleAnalyticsListAccountsOperator`
+
+List Google Ads Links
+^^^^^^^^^^^^^^^^^^^^^
+
+To list Google Ads links you can use the
```

Comment:
I think it would also be great to describe that "list" means store as xcom.

Reply:
I agree.